.travis.yml

```diff
@@ -69,6 +69,157 @@ jobs:
       after_success: false
       after_failure: false
       if: branch = master AND type = push
+
+    - language: python
+      stage: test
+      python: 3.4
+      addons:
+        apt:
+          packages: clang-3.4
+
+    - language: python
+      python: 3.5
+      addons:
+        apt:
+          packages: clang-3.4
+
+    - language: node_js
+      node_js: 4
+    - language: node_js
+      node_js: 5
+    - language: node_js
+      node_js: 6
+
+    - language: ruby
+      rvm: 2.2
+    - language: ruby
+      rvm: 2.1
+    - language: ruby
+      rvm: 2.0
+
+    - language: haskell
```

Comment:
this job isnt working yet in the new PR https://github.com/coala/coala-bears/pull/2910

the `TRAVIS_GHC_ROOT=/opt/ghc` env setting below might be relevant.